client/components/LearnerViewDirective.js

```diff
@@ -37,99 +37,103 @@ tie.directive('learnerView', [function() {
                   <div class="tie-dot tie-dot-1" ng-class="{'night-mode': isInDarkMode}"></div>
                   <div class="tie-dot tie-dot-2" ng-class="{'night-mode': isInDarkMode}"></div>
                   <div class="tie-dot tie-dot-3" ng-class="{'night-mode': isInDarkMode}"></div>
                 </div>
-              </div>
-              <div class="tie-coding-window">
-                <div class="tie-lang-terminal">
-                  <div class="tie-coding-terminal">
-                    <ui-codemirror ui-codemirror-opts="codeMirrorOptions"
-                        ng-model="code"
-                        ng-change="autosave()"
-                        class="tie-codemirror-container"></ui-codemirror>
-                  </div>
-                  <select class="tie-lang-select-menu" name="lang-select-menu">
-                    <option value="Python" selected>Python</option>
-                  </select>
-                  <select class="tie-question-set-select" name="question-set-select"
-                          ng-change="changeQuestionSet(currentQuestionSetId)" ng-model="currentQuestionSetId"
-                          ng-options="i.questionSetId as i.questionSetId for i in questionSetIds">
-                    <option style="display: none" value="">Question Set</option>
-                  </select>
-                  <select class="tie-theme-select" name="theme-select"
-                          ng-change="changeTheme(theme)" ng-model="theme"
-                          ng-options="i.themeName as i.themeName for i in themes">
-                    <option style="display: none" value="">Theme</option>
-                  </select>
-                  <button class="tie-code-reset" name="code-reset"
-                      ng-click="resetCode()">
-                    Reset Code
-                  </button>
-                  <div class="tie-code-auto-save" ng-class="{'night-mode': isInDarkMode}" ng-show="autosaveTextIsDisplayed">
-                    Saving code...
-                  </div>
-                  <button class="tie-run-button"
-                      ng-class="{'active': !nextButtonIsShown}"
-                      ng-click="submitCode(code)"
-                      ng-disabled="nextButtonIsShown">
-                    Run
-                  </button>
-                  <div class="tie-next-curtain-container"
-                      ng-if="nextButtonIsShown">
-                    <div class="tie-next-curtain"></div>
-                    <div class="tie-arrow-highlighter"></div>
-                    <div ng-click="showNextTask()" class="tie-next-arrow">
-                      <span class="tie-next-button-text">Next</span>
-                    </div>
-                  </div>
                 </div>
               </div>
+              <select class="tie-select-menu" name="question-set-select"
+                      ng-change="changeQuestionSet(currentQuestionSetId)" ng-model="currentQuestionSetId"
```

Comment:
For continuation lines of HTML tags only, I prefer to have the attrs aligned as they are now (i.e. left edge of "ng-change" aligned with left edge of "class"). So I think the current version of lines 96-97 is fine in that regard. This helps distinguish from actual nesting of HTML tags within others, which we do with 2-space indentation.

The reason for using 2-space rather than 4-space is that HTML tags can get pretty nested. So I would lean towards the former in that case.

Does the above sound OK?